REPORT.md

```diff
@@ -341,16 +660,723 @@ vainfo: Supported profile and entrypoints
+|MIP_A_HHI_3|✔️|
+|MIP_B_HHI_3|✔️|
+|MMVD_A_SAMSUNG_3|⌛|
+|MNUT_A_Nokia_3|☠|
+|MNUT_B_Nokia_2|☠|
+|MPM_A_LGE_3|⌛|
+|MRLP_A_HHI_2|✔️|
+|MRLP_B_HHI_2|✔️|
+|MTS_A_LGE_4|✔️|
+|MTS_B_LGE_4|✔️|
+|MTS_LFNST_A_LGE_4|✔️|
+|MTS_LFNST_B_LGE_4|✔️|
+|MVCOMP_A_Sharp_2|✔️|
+|OLS_A_Tencent_4|❌|
+|OLS_B_Tencent_4|❌|
+|OLS_C_Tencent_4|❌|
+|OPI_A_Nokia_1|✔️|
+|OPI_B_Nokia_2|❌|
+|PALETTE_A_Alibaba_2|❌|
+|PALETTE_B_Alibaba_2|☠|
+|PALETTE_C_Alibaba_2|☠|
+|PALETTE_D_Alibaba_2|☠|
+|PALETTE_E_Alibaba_2|❌|
+|PDPC_A_Qualcomm_3|✔️|
+|PDPC_B_Qualcomm_3|✔️|
+|PDPC_C_Qualcomm_2|✔️|
+|PHSH_B_Sharp_1|✔️|
+|PMERGE_A_MediaTek_1|✔️|
+|PMERGE_B_MediaTek_1|✔️|
+|PMERGE_C_MediaTek_1|✔️|
+|PMERGE_D_MediaTek_1|✔️|
+|PMERGE_E_MediaTek_1|✔️|
+|POC_A_Nokia_1|✔️|
+|POUT_A_Sharplabs_2|✔️|
+|PPS_A_Bytedance_1|⌛|
+|PPS_B_Bytedance_1|✔️|
+|PPS_C_Bytedance_1|✔️|
+|PQ_A_Dolby_1|✔️|
+|PROF_A_Interdigital_3|✔️|
+|PROF_B_Interdigital_3|✔️|
+|PSEXT_A_Nokia_2|✔️|
+|PSEXT_B_Nokia_2|✔️|
+|QTBTT_A_MediaTek_4|✔️|
+|QUANT_A_Huawei_2|✔️|
+|QUANT_B_Huawei_2|✔️|
+|QUANT_C_Huawei_2|✔️|
+|QUANT_D_Huawei_4|✔️|
+|QUANT_E_Interdigital_1|✔️|
+|RAP_A_HHI_1|✔️|
+|RAP_B_HHI_1|✔️|
+|RAP_C_HHI_1|✔️|
+|RAP_D_HHI_1|✔️|
+|RPL_A_ERICSSON_2|✔️|
+|RPR_A_Alibaba_4|❌|
+|RPR_B_Alibaba_3|❌|
+|RPR_C_Alibaba_3|❌|
+|RPR_D_Qualcomm_1|❌|
+|SAO_A_SAMSUNG_3|✔️|
+|SAO_B_SAMSUNG_3|✔️|
+|SAO_C_SAMSUNG_3|✔️|
+|SBT_A_HUAWEI_2|✔️|
+|SbTMVP_A_Bytedance_3|✔️|
+|SbTMVP_B_Bytedance_3|✔️|
+|SCALING_A_InterDigital_1|✔️|
+|SCALING_B_InterDigital_1|✔️|
+|SCALING_C_InterDigital_1|✔️|
+|SDH_A_Dolby_2|✔️|
+|SLICES_A_HUAWEI_2|✔️|
+|SMVD_A_HUAWEI_2|✔️|
+|SPATSCAL444_A_Qualcomm_2|❌|
+|SPATSCAL_A_Qualcomm_3|❌|
+|SPS_A_Bytedance_1|⌛|
+|SPS_B_Bytedance_1|✔️|
+|SPS_C_Bytedance_1|✔️|
+|STILL444_A_KDDI_1|✔️|
+|STILL444_B_ERICSSON_1|✔️|
+|STILL_A_KDDI_1|✔️|
+|STILL_B_ERICSSON_1|✔️|
+|SUBPIC_A_HUAWEI_3|✔️|
+|SUBPIC_B_HUAWEI_3|✔️|
+|SUBPIC_C_ERICSSON_1|☠|
+|SUBPIC_D_ERICSSON_1|☠|
+|SUBPIC_E_MediaTek_1|✔️|
+|SUFAPS_A_HHI_1|✔️|
+|TEMPSCAL_A_Panasonic_4|✔️|
+|TEMPSCAL_B_Panasonic_5|⌛|
+|TEMPSCAL_C_Panasonic_3|✔️|
+|TILE_A_Nokia_2|✔️|
+|TILE_B_Nokia_2|✔️|
+|TILE_C_Nokia_2|✔️|
+|TILE_D_Nokia_2|✔️|
+|TILE_E_Nokia_2|✔️|
+|TILE_F_Nokia_2|✔️|
+|TILE_G_Nokia_2|✔️|
+|TMVP_A_Chipsnmedia_3|✔️|
+|TMVP_B_Chipsnmedia_3|✔️|
+|TMVP_C_Chipsnmedia_3|✔️|
+|TMVP_D_Chipsnmedia_3|✔️|
+|TRANS_A_Chipsnmedia_2|✔️|
+|TRANS_B_Chipsnmedia_2|✔️|
+|TRANS_C_Chipsnmedia_4|✔️|
+|TRANS_D_Chipsnmedia_4|✔️|
+|TREE_A_HHI_3|✔️|
+|TREE_B_HHI_3|⌛|
+|TREE_C_HHI_3|⌛|
+|VIRTUAL_A_MediaTek_3|✔️|
+|VIRTUAL_B_MediaTek_3|✔️|
+|VPS_A_INTEL_3|❌|
+|VPS_B_ERICSSON_1|❌|
+|VPS_C_ERICSSON_1|☠|
+|WP_A_InterDigital_3|✔️|
+|WP_B_InterDigital_3|✔️|
+|WPP_A_Sharp_3|✔️|
+|WPP_B_Sharp_2|✔️|
+|WRAP_A_InterDigital_4|✔️|
+|WRAP_B_InterDigital_4|✔️|
+|WRAP_C_InterDigital_4|✔️|
+|WRAP_D_InterDigital_4|✔️|
+|-|-|
+|TOTAL|217/282|
+
+
+### H.265/HEVC Nvidia
+
+`./fluster.py run -d JCT-VT-H.265 GStreamer-H.265-VAAPI-Gst1.0 FFmpeg-H.265-VAAPI GStreamer-H.265-NVDEC-Gst1.0 FFmpeg-H.265-VDPAU FFmpeg-H.265 -ts JCT-VC-HEVC_V1 -s`
```

Comment:
I would remove all VAAPI related decoders from the list and the results. Supposedly VAAPI does not support Nvidia, internally FFmpeg and GStreamer might be using your Intel GPU without us knowing.